### PR TITLE
Set inter font for all validation tests

### PR DESCRIPTION
**Pull request by yiqunw700**
This should fix issue #1186 that github workflow generates consistent screenshot diffs for variant tests. The verifyRoborazzi check are not failing any more.

I believe this is similar to the interaction states not get cleared. On the local machine, it seems tests from different kt files are running on different processes but the github flow somewhat runs different kt test files on the same process.

Right now some of the tests have the font set and others not. So I am setting font to all tests so the screenshot are generated using the same font no matter if it shares a process with others or not.

But further more, we might want to look into why and how to isolate tests?? I haven't found anything how to do that for Robolectric though.

### CODE REVIEW

**Comment by timothyfroehlich** [comment]
Looks good to me, but could you update the description with the reasoning for this change?

File: integration-tests/validation/src/testFixtures/java/com/android/designcompose/testapp/validation/InterFontTestRule.java
Comment (timothyfroehlich) [inline integration-tests/validation/src/testFixtures/java/com/android/designcompose/testapp/validation/InterFontTestRule.java:1]: Could you move this to be part of the app-common package? I think it's likely we'll re-use this rule.

**Review by timothyfroehlich — APPROVED** [review]
Great catch that it's the font! And yeah, it's super odd that the test process seems to be shared. Perhaps there's some environment variable that's set on the runners to try to combine the processes to make them run faster.

File: integration-tests/validation/src/testFixtures/java/com/android/designcompose/testapp/validation/InterFontTestRule.java
Comment (timothyfroehlich) [reply integration-tests/validation/src/testFixtures/java/com/android/designcompose/testapp/validation/InterFontTestRule.java:1]: If you can and it works then go ahead and push that change and submit.

**Comment by rylin8** [comment]
Nice find! Not sure what triggered it to behave differently but seems this should be done anyway.

**Review by rylin8 — APPROVED** [review]

File: integration-tests/validation/src/testFixtures/java/com/android/designcompose/testapp/validation/InterFontTestRule.java
Comment (yiqunw700) [reply integration-tests/validation/src/testFixtures/java/com/android/designcompose/testapp/validation/InterFontTestRule.java:1]: Done. Move to app common package.

**Pull request merged**